docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,251 @@ sidebar_label: Wikidata
+*   A large number of potential types to reconcile against
+*   Previewing and viewing entities
+*   Suggesting entities, types, and properties
+*   Augmenting your dataset with more information pulled from Wikidata 
 
+You can find the documentation and further resources [here](https://wikidata.reconci.link/).
 
-*   Edit Wikidata schema
-*   Manage Wikidata account
-*   Import schema
-*   Export schema
-*   Uploading edits to Wikidata
-*   QuickStatements export
-*   Schema alignment
-*   Issue detection
+For the most part, Wikidata reconciliation behaves the same way other reconciliation processes do, but there are a few processes and features specific to Wikidata. 
 
+### Language settings
 
-## More resources
+You can install a version of the Wikidata reconciliation service that uses your language.
 
-[https://wdreconcile.toolforge.org/](https://wdreconcile.toolforge.org/) 
+First, you need your language code: this is the two-letter code in the domain name of your Wikipedia (for instance "fr" if your Wikipedia is [https://fr.wikipedia.org/wiki/](https://fr.wikipedia.org/wiki/)).
 
-[https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine](https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine) 
+Then, open the reconciliation dialog and click <span class="menuItems">Add Standard Service</span>. The URL is `https://openrefine-wikidata.toolforge.org/fr/api` where "fr" is replaced by your language code.
+
+When reconciling using this interface, items and properties will be displayed in your language if a translation is available. The matching score of the reconciliation is not influenced by your choice of language: items are matched by considering all labels and keeping the best possible match. So the language of your dataset is irrelevant to the choice of the language for the reconciliation interface.
+
+### Restricting matches by type
+
+In Wikidata, types are items themselves. For instance, the [university of Ljubljana (Q1377)](https://www.wikidata.org/wiki/Q1377) has type [public university (Q875538)](https://www.wikidata.org/wiki/Q875538), which is witnessed by a statement on [Q1377](https://www.wikidata.org/wiki/Q1377) using the [instance of (P31)](https://www.wikidata.org/wiki/Property:P31) property.
+
+Types are organized into an ontology, by specifying which types are subclasses of the others, using the [subclass of (P279)](https://www.wikidata.org/wiki/Property:P279) property. For instance, [public university (Q875538)](https://www.wikidata.org/wiki/Q875538) is a subclass of [university (Q3918)](https://www.wikidata.org/wiki/Q3918).
+
+One useful tool to visualize the subclasses of a given type is the [Wikidata Graph Builder](https://angryloki.github.io/wikidata-graph-builder/). For instance, here are [all the subclasses of academic institution (Q4671277)](https://angryloki.github.io/wikidata-graph-builder/?property=P279&item=Q4671277&mode=reverse).
```

Comment:
This seems like quite a lot of detail - do we want to go into this level of detail about wikidata ontology structure?